<!DOCTYPE html>
<html>
<body>
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
<script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn" crossorigin="anonymous"></script>

<style type="text/css"></style>

<br>
<br>
<br>
<div class="container">

<h1> Analysis of differential gene expression signatures induced by HIV infection of podocytes </h1>

<h2> Overview </h2>

<p>The following notebook contains the results of a computational analysis of gene expression signatures generated by HIV infection of podocytes.</p>

<p>The analysis was performed on two different datasets:</p>
<ol>
<li>
	A dataset consisting of 21 samples of <b>primary podocytes</b>, isolated from different patients.  Samples are divided in three groups:
	<ul>
		<li><i>HIV-infected</i> (7 samples)</li>
		<li><i>GFP-infected controls</i> (6 samples)</li>
		<li><i>Untreated controls</i> (8 samples)</li>
	</ul>
</li>
<br>
<li>
	A dataset consisting of 14 samples of <b>podocyte-derived cell lines</b>.  Samples were processed in three batches, and are divided in two main groups:
	<ul>
		<li><i>Untreated</i> controls (3 samples)</li>
		<li><i>HIV-infected</i> samples (11 samples), separated in the following timepoints:</li>
			<ul>
				<li><i>6 hours</i> post-infection (3 samples)</li>
				<li><i>12 hours</i> post-infection (2 samples)*</li>
				<li><i>24 hours</i> post-infection (2 samples)*</li>
				<li><i>48 hours</i> post-infection (3 samples)</li>
		</li>
	</ul>
</li>
</ol>
<p style="font-size: small;">*In order to have a minimum of 3 samples necessary to compute differential expression, the two timepoints were merged in a single 12-24 hour timepoint.</p>

<h2> Key Findings </h2>

<ul>
	<li><b>Data is significantly affected by batch effects</b>, whereby most of the variance in the data is related to the batch or patient of origin.  Computational methods partly assist in recovering the original biological variance.</li>
	<li>HIV-infected podocytes in both datasets display <b>overexpression</b> of genes involved in the following pathways:
		<ul>
			<li><b>defense response and negative regulation of virus replication</b></li>
			<li><b>immune response and cytokine signaling</b></li>
			<li><b>TNF signaling</b></li>
		</ul>
	</li>
	<li>HIV-infected podocytes in both datasets display <b>underexpression</b> of genes associated to the <b>Polycomb-group (PcG) family</b>, including targets of the following genes:
		<ul>
			<li><b>SUZ12</b></li>
			<li><b>EED</b></li>
			<li><b>KDM2B</b></li>
			<li><b>RNF2</b></li>
		</ul>
	</li>
	<li>The <b>sterol biosynthesis</b> and <b>steroid hormone response</b> pathways are commonly dysregulated in both datasets, showing both overexpression and underexpression of related terms.</li>
	<li>A number of small molecules which reproducibly mimic and reverse the HIV infection signatures are identified: </li>
		<ul>
			<li><b>Mimic</b>: <i>CGP-60474</i>, <i>mitoxantrone</i>, <i>alvocidib</i>, and others.</li>
			<li><b>Reverse</b>: <i>Emetine Dihydrochloride Hydrate</i>, <i>QL-XII-47</i>, and others.</li>
		</ul>
	</li>
</ul>

<h2> Computational Methods </h2>

<p>Each dataset was analyzed using the following pipeline:</p>

<ol>
	<li><b>Normalize raw expression</b> levels using DESeq2's Variance Stabilizing Transformation method (<a href="https://rdrr.io/bioc/DESeq2/man/varianceStabilizingTransformation.html">1</a>).</li>
	<li>Perform a <b>Principal Components Analysis</b> (PCA) to assess separation between samples.</li>
	<li><b>Remove batch effects</b> from the data using the ComBat algorithm (<a href="https://www.bu.edu/jlab/wp-assets/ComBat/Abstract.html">2</a>).</li>
	<li>Calculate a <b>differential gene expression signature</b> using the Characteristic Direction method (<a href="http://www.maayanlab.net/CD/">3</a>).</li>
	<li>Perform an <b>enrichment analysis</b> of the top 500 most overexpressed and underexpressed genes using Enrichr (<a href="http://amp.pharm.mssm.edu/Enrichr/">4</a>)</li>
	<li>Identify <b>small molecule perturbations</b> which <b>mimic or reverse</b> the gene expression signatures identified in step 4. (<a href="http://amp.pharm.mssm.edu/L1000CDS2/#/index">5</a>)</li>
</ol>

<h2> Results </h2>

<h3 id="pca-analysis"> 1. PCA Analysis </h3>

<p>In order to assess the separation between samples in an unbiased manner, a Principal Components Analysis (PCA) was performed on both datasets.</p>
<p>The PCA analysis on the <b>primary podocyte dataset</b> (<i>Figure 1A</i>) shows that <b>similarly treated samples do not clearly cluster together</b>. This might be due to the presence of batch effects, as some samples coming from the same patient cluster together despite being treated differently. The same analysis was repeated after batch effect removal by ComBat (<i>Figure 1B</i>).  The algorithm was able to correct the dataset in order to ensure a better separation between samples which underwent similar biological treatment .</p>

<p>The PCA analysis on the <b>cell line dataset</b> (<i>Figure 2A</i>) shows that <b>samples strongly cluster by batch</b>. This batch effect overrides the variance contributed by the HIV infection treatment.  The same analysis was repeated after batch effect removal by ComBat (<i>Figure 2B</i>).  As above, the algorithm was able to correct the dataset in order to ensure a somewhat better separation between samples which underwent similar biological treatment .</p>

<h3 id="enrichment-analysis"> 2. Enrichment Analysis </h3>

<p>In order to investigate the <b>commonly dysregulated pathways following HIV infection podocytes</b>, an enrichment analysis was performed on the differential gene expression signatures calculated comparing untreated samples to infected samples.</p>

<p>The enrichment analysis in the <b>primary podocyte dataset</b> reveals the following: </p>
<ul>
	<li><b>Upregulation</b> of genes involved in pathways relative to <b>immune response</b> and <b>inflammation</b>.  Enriched terms include <i>defense response to virus</i>, <i>cytokine-mediated signaling pathway</i>, and <i>negative regulation of viral genome replication</i></li>
	<li><b>Downregulation</b> of genes involved in pathways relative to <b>cell proliferation</b> and <b>cell migration</b>.  Enriched terms include <i>regulation of kidney development</i>, <i>leukocyte transendothelial migration</i>, and <i>stress fiber</i></li>
	<li><b>Dysregulation</b> of genes involved in <b>sterol biosynthesis and response</b>. Upregulated genes are enriched for terms such as <i>cholesterol biosynthetic process</i> and <i>abnormal lipid homeostasis</i>, while downregulated genes are enriched for terms such as <i>response to steroid hormone</i> and <i>response to estrogen</i>.
</ul>

<p>Full results available <a href="#enrichment-results">here</a>.</p>


<p>The enrichment analysis in the <b>cell line dataset</b> reveals the following: </p>
<ul>
	<li><b>Upregulation</b> of genes involved in <b>defense response to virus</b>, and <b>TNF signaling</b> at the 6 hour timepoint, reaching a peak of enrichment at the 48 hour timepoint.</li>
	<li><b>Upregulation</b> of genes involved in <b>steroid binding</b> at the 6 hour timepoint, which decreases in the latter timepoints.</li>
	<li><b>Downregulation</b> of genes involved in pathways relative to <b>cell metabolism</b> and the <b>mitochondrion</b> at the 6 hour timepoint, decreasing in the latter timepoints.  Enriched terms include <i>mitochondrial respiratory chain complex I</i>, <i>NADH dehydrogenase activity</i>, and <i>ubiquinone biosynthetic process</i></li>
	<li><b>Downregulation</b> of genes involved in <b>kinase inhibition</b>, reaching a peak at the 12 hour timepoint.  Enriched terms include <i>kinase regulator activity</i>, and <i>cyclin-dependent protein serine/threonine kinase regulator activity</i>.</li>
</ul>

<p>Full results available <a href="#enrichment-results">here</a>.</p>

<p>A <b>comparison of enrichment results</b> between the primary podocyte and the cell-line 48 hour HIV infection signatures reveal several commonly enriched pathways:</p>
<ul>
	<li><b>Upregulation</b> of genes involved in <b>defense response</b>, <b>negative regulation of viral genome replication</b>, and <b>TNF signaling</b>.</li>
	<li><b>Downregulation</b> of genes associated to the <b>Polycomb-group (PcG) family</b>, including targets of <b>SUZ12</b>, <b>EED</b>, <b>KDM2B</b>, and <b>RNF2</b>.</li>
</ul>

<h3 id="l1000cds2-analysis"> 3. Small Molecule Identification </h3>

<h4>Overview</h4>

<p>In order to identify <b>small molecules which mimic or reverse HIV infection signatures</b>, a query was performed using L1000CDS<sup>2</sup> with the following differential gene expression signatures:
<ul>
	<li>The <b>primary podocyte</b> signature, <i>HIV-infected vs. GFP-infected</i></li>
	<li>The <b>podocyte cell line</b> signatures, <i>HIV-infected vs. uninfected controls</i> at the following timepoints:
		<ul>
			<li><i>6 hours post-infection</i></li>
			<li><i>12-24 hours post-infection</i></li>
			<li><i>48 hours post-infection</i></li>
		</ul>
	</li>
</ul>

<p>For each of these, the top 50 most similar and most opposite signatures were identified amongst the library of small molecule perturbations from L1000.  Drugs were then ranked according to how frequently they appeared within the top 50 signatures.</p>

<h4>Results</h4>

<p>The analysis revealed the following results:</p>
<ul>
	<li><b>CGP-60474</b> (<a href="https://pubchem.ncbi.nlm.nih.gov/compound/644215">link</a>), a dual CDK1/CDK2 inhibitor, reproducibly mimics the later timepoints of cell line HIV infection (12, 24, and 48 hours), as well as the HIV infection signature in primary podocytes.  The function of the compound is not well characterized.</li>
	<li><b>Mitoxantrone</b> (<a href="https://pubchem.ncbi.nlm.nih.gov/compound/4212">link</a>), a topoisomerase inhibitor, is ranked within the top mimics for all the HIV infection signatures.
		<ul>
			<li>It is an <b>FDA-approved drug</b> used for treatment of multiple sclerosis, prostate cancer and certain types of leukemia.</li>
			<li> 
				It has been shown to <b>inhibit viral replication and assembly</b> in independent studies (<a href="https://www.ncbi.nlm.nih.gov/pubmed/22182595">Altmann et al., Antiviral Res. 2012</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/17928345">Deng et al., J Virol. 2007</a>)
			</li>
		</ul>
	</li>
	<li><b>Emetine Dihydrochloride Hydrate</b> (<a href="https://pubchem.ncbi.nlm.nih.gov/compound/emetine">link</a>), a natural plant alkaloid, is ranked within the top reversing drugs for all HIV infection signatures.  
		<ul>
			<li>
				Emetine has been shown to <b>inhibit HIV-1 replication</b> by interfering with reverse transcriptase activity (<a href="https://www.ncbi.nlm.nih.gov/pubmed/26111177">Chaves et al., Molecules 2015</a>).
			</li>
			<li>
				It has also been shown to display <b>antiviral activity against human Cytomegalovirus</b> (<a href="https://www.ncbi.nlm.nih.gov/pubmed/27336364">Mukhopadhyay et al, PLoS Pathog. 2016</a>) and <b>Dengue virus</b> (<a href="https://www.omicsonline.org/open-access/antiviral-activity-of-emetine-dihydrochlorideagainst-dengue-virus-infection-jaa-1000009.php?aid=74546">Low et al., J. Antivirals & Antiretrovirals 2009</a>)
			</li>
		</ul>
	</li>
	<li><b>QL-XII-47</b> (<a href="https://lincs.hms.harvard.edu/db/sm/10077-101-1/">link</a>), a kinase inhibitor, is ranked within the top reversing drugs for all but one HIV infection signatures.  
		<ul>
			<li>
				It has been shown to act as a <b>broad-spectrum antiviral agent</b> against dengue and other RNA viruses (<a href="http://pubs.acs.org/doi/abs/10.1021/acsmedchemlett.7b00008">Liang et al., ACS Med. Chem. Lett. 2017</a>).
			</li>
		</ul>
	</li>
</ul>

<p>Full results available <a href="#l1000cds2-results">here</a>.</p>


<h2 id="enrichment-results"> Full Results </h2>
<h3>Enrichment Analysis</h3>
<p> Full results for the enrichment analyses are available at the following links:</p>
<p><b>Primary Podocyte Analysis</b></p>

<p><b>Cell Line Analysis</b></p>

<p><b>Interactive Heatmap for Cell line enrichment results</b></p>
<p><b>Upregulated</b></p>

<p><b>Downregulated</b></p>

<h3>Small Molecule Search</h3>
<p id="l1000cds2-results"> Full results for the enrichment analyses are available at the following links:</p>
<p><b>Primary Podocyte Analysis</b></p>

<p><b>Cell Line Analysis</b></p>



</div>

</body>
</html>

